src/selection/merge.js

```diff
@@ -1,6 +1,7 @@
 import {Selection} from "./index.js";
 
-export default function(selection) {
+export default function(context) {
+  var selection = (context && context.selection) ? context.selection() : context;
```

Comment:
Or we could be fancy and use the new optional chaining operator.
```suggestion
  var selection = context?.selection ? context.selection() : context;
```

Reply:
The _context_ argument is required (it’ll throw an Error if not specified), so it doesn’t seem necessary to test here. This should work:

```suggestion
  var selection = context.selection ? context.selection() : context;
```

This is what [d3-axis does](https://github.com/d3/d3-axis/blob/4fc4f2bd287de20c4fc5f1da447ae1194d3acca6/src/axis.js#L52), too.

Reply:
Nice. Thanks!